minecraft-server/Dockerfile

```diff
@@ -1,25 +1,26 @@
-FROM java:8
+FROM alpine
 
 MAINTAINER itzg
 
-ENV APT_GET_UPDATE 2016-04-23
-RUN apt-get update
-
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
-  imagemagick \
-  lsof \
-  nano \
-  sudo \
-  vim \
-  jq \
-  && apt-get clean
-
-RUN useradd -s /bin/false --uid 1000 minecraft \
+RUN echo "http://dl-3.alpinelinux.org/alpine/v3.5/community/" >> /etc/apk/repositories &&\
+        apk update && \
+        apk add \
+          openjdk8-jre-base \
+          openssl \
+          imagemagick \
+          lsof \
+          su-exec \
+          bash \
+          git \
+          jq &&\
```

Comment:
Please also add `ca-certificates` so that x509 certs resolve while restify accesses https URLs.